docs/spl-docs/evaluation-functions/trig-hyperbolic-functions.md

```diff
@@ -0,0 +1,189 @@
+# Trigonometric and Hyperbolic functions
+
+The following list contains the functions that you can use to calculate trigonometric and hyperbolic values.
+
+## acos(x)
+
+This function computes the arc cosine of `x`, in the interval `[0,pi]` radians.
```

Comment:
What happens if `x` is outside the domain?

Reply:
We are throwing an error, but that error is not shown on the UI. It just shows empty results.